### ISSUE TITLE
Expose the Shadow version number

### ISSUE THREAD

**Issue by Daniel-t** [aws/aws-iot-device-sdk-js#98]
It appears that a deliberate design decision was made to prevent the shadow's version number from being accessible from consuming applications (see comments pasted below), however I have a use case which would benefit from the application having access to the version, e.g. a simple counter of state changes.
```
      // Remove the properties 'clientToken' and 'version' from the stateObject;
      // these properties are internal to this class.
```

Can this be removed in a future version to allow applications to use the version number?  Is there a way to access the version number that I'm missing?

I've solved my immediate problem by deleting the following line from my local copy, but this is far from ideal. 
`delete stateObject.version;`

**Comment by fengsongAWS** [comment]
Hi @Daniel-t ,
Thanks for your interest in AWS IoT Device SDK. 
We will look into more of use cases of shadow operations and we can add it in the future release.

Fengyi

**Comment by fengsongAWS** [comment]
Issue is fixed in v2.2.0
